web/docs/understand/operators/transformations/flatten.md

```diff
@@ -0,0 +1,57 @@
+# flatten
+
+Flattens nested data structures.
+
+## Synopsis
+
+```
+flatten
+```
+
+## Description
+
+The `flatten` operator removes any nested lists or records and folds them into
+the top-level structure of event. It removes null values inside lists.
+
+## Examples
+
+Consider the following data:
+
+```json
+{
+  "src_ip": "147.32.84.165",
+  "src_port": 1141,
+  "dest_ip": "147.32.80.9",
+  "dest_port": 53,
+  "event_type": "dns",
+  "dns": {
+    "type": "query",
+    "id": 553,
+    "rrname": "irc.freenode.net",
+    "rrtype": "A",
+    "tx_id": 0,
+    "grouped": {
+      "A": ["tenzir.com", null]
+    }
+  }
+}
+```
+
+The `flatten` operator removes any nesting from the data:
```

Comment:
```suggestion
The `flatten` operator removes any nesting from the data by joining nested fields with a `.`:
```

It would be nice to tell the user *how* the flattening takes place. Perhaps already even earlier in the description.